public/script.js

```diff
@@ -22,8 +22,19 @@ const DEFAULTS = {
 };
 
 let map;
+const validDates = [];
 /** Main function run to start animation */
 async function main() {
+  await fetch('https://nsidc.org/api/mapservices/NSIDC/wms?service=wms&version=1.1.1&request=GetCapabilities').then(function(response) {
+    return response.text();
+  }).then(function(text) {
+    const parser = new DOMParser();
+    const xmlDoc = parser.parseFromString(text, 'text/xml');
+    const extents = xmlDoc.getElementsByTagName('Extent');
+    for (i = 0; i <= 7; i++) {
+      validDates.push(extents[i].childNodes[0].nodeValue.split(','));
```

Comment:
Comment the meaning of the loop and the reasons for looping 0 through 7 (should be replaced regardless).